src/components/CommentForm/CommentForm.stories.tsx

```diff
@@ -66,6 +66,16 @@ export const Default = () => (
 );
 Default.story = { name: 'default' };
 
+export const Error = () => (
+	<CommentForm
+		shortUrl={'/p/g8g7v'}
+		pillar={Pillar.News}
+		user={aUser}
+		onAddComment={(comment) => {}}
+	/>
+);
+Error.story = { name: 'Error' };
```

Comment:
Can we change the name to provide more info, similar to `Active` below it? 
```suggestion
Error.story = { name: 'form with errors' };
```